orca-webhook/src/main/groovy/com/netflix/spinnaker/orca/webhook/tasks/CreateWebhookTask.groovy

```diff
@@ -47,7 +49,13 @@ class CreateWebhookTask implements RetryableTask {
     def customHeaders = stage.context.customHeaders
     boolean waitForCompletion = (stage.context.waitForCompletion as String)?.toBoolean()
 
-    def response = webhookService.exchange(method, url, payload, customHeaders)
+    def response
+    try {
+       response = webhookService.exchange(method, url, payload, customHeaders)
+    } catch (Exception e) {
```

Comment:
I thought `restTemplate.exchange()` did not throw exceptions on server side errors, but I see now that `DefaultResponseErrorHandler` is in fact doing that on 4xx and 5xx status codes. Sorry.
But this means some of the logic below should be refactored as well.